test/staging/upsert/Map/getOrInsert/append-new-values-normalizes-zero-key.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2024 Jonas Haukenes. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: proposal-upsert
+description: |
+  Append a new value in the map normalizing +0 and -0.
+info: |
+  Map.prototype.getOrInsert ( key , value )
+
+  ...
+  3. Set key to CanonicalizeKeyedCollectionKey(key).
+  4. For each Record { [[Key]], [[Value]] } p of M.[[MapData]], do
+    a. If p.[[Key]] is not empty and SameValue(p.[[Key]], key) is true, return p.[[Value]].
+  5. Let p be the Record { [[Key]]: key, [[Value]]: value }.
+  6. Append p to M.[[MapData]].
+  ...
+features: [Symbol]
+flags: [noStrict]
+---*/
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
possibly not? (ditto 50+ times below)

Reply:
Yep, that looks like an artifact of the export process that's not present in the Firefox copies of the tests.

Reply:
Why are there two lines? I'm confused by these having 2015 authorship.

Reply:
A lot of the tests were based on existing Map/WeakMap tests, so they kept the original copyright statement.